carmm/analyse/counterpoise_onepot.py

```diff
@@ -0,0 +1,53 @@
+def create_counterpoise_input(complex_struc=None, a_id=None, b_id=None, symbol_not_index=True):
```

Comment:
Can you add an example to the regression test with some assertions?

Reply:
Will do this soon.